tests/mock_network/utils.rs

```diff
@@ -761,8 +763,11 @@ pub fn verify_section_invariants_between_nodes(nodes: &[TestNode]) {
     };
     let mut sections: BTreeMap<Prefix<XorName>, NodeSectionInfo> = BTreeMap::new();
 
-    for node in nodes.iter() {
-        let our_prefix = unwrap!(node.inner.our_prefix());
+    for node in nodes.iter().filter(|node| node.inner.is_elder()) {
+        let our_prefix = match node.inner.our_prefix() {
+            Some(pfx) => pfx,
+            None => continue,
+        };
```

Comment:
I think this can be replaced with just `node.our_prefix()` (which unwraps internally), because we are iterating only elders and so are guaranteed to have prefix.

Reply:
Fixed with https://github.com/maidsafe/routing/pull/1959